generated/java/proto-google-longrunning-v1/build.gradle

```diff
@@ -0,0 +1,157 @@
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+}
+
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'signing'
+
+description = 'PROTO library for proto-google-longrunning-v1'
+group = "com.google.api.grpc"
+version = "0.1.11"
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+repositories {
+  mavenCentral()
+  mavenLocal()
+}
+
+dependencies {
+  compile "com.google.protobuf:protobuf-java:3.2.0"
+  compile "com.google.api:api-common:1.1.0"
+  compile "com.google.api.grpc:proto-google-common-protos:0.1.11"
```

Comment:
It looks like there are more places that could have the project dependency (iam too)